debian/changelog

```diff
@@ -0,0 +1,6 @@
+hdfmonkey (0.4.1) UNRELEASED; urgency=medium
```

Comment:
This says "UNRELEASED" so that it can't be uploaded by mistake to Debian proper.